src/tests/ftest/util/server_utils_params.py

```diff
@@ -441,6 +441,7 @@ class EngineYamlParameters(YamlParameters):
             "D_LOG_FILE_APPEND_PID=1",
             "DAOS_POOL_RF=4",
             "CRT_EVENT_DELAY=1",
+            "DAOS_VOS_AGG_GAP=25",
```

Comment:
We need to consider whether this actually belongs in test code because it means we will not be testing what is used in production